fetools-app/src/components/TableGeneratorComponents/TableGenerator.jsx

```diff
@@ -0,0 +1,55 @@
+import { handleEditCells, getCellStyle } from './TableGeneratorFN';
+
+export const TableGenerator = ({ tableConfig, setTableConfig }) => {
+  return (
+    <div>
+      <div
+        className={`overflow-hidden`}
+        style={{
+          borderRadius: `${tableConfig.borderRounding}px`,
+          width: `${tableConfig.tableWidth}px`,
+          border: `${tableConfig.borderWidth}px ${tableConfig.borderStyle} ${tableConfig.borderColor}`,
+        }}
+      >
+        <table>
+          {tableConfig.dimensions.map((row, idxRow) => (
+            <tr key={idxRow}>
+              {row.map((col, idxCols) => {
+                const isHeaderRow = idxRow === 0;
+
+                return (
+                  <td
+                    style={getCellStyle(idxRow, tableConfig)}
+                    className="min-h-8 border-black"
+                    key={idxCols}
+                  >
+                    <input
```

Comment:
Using <input> here causes the table's layout to be different than expected. We need to use an editable state to toggle between edit mode and normal mode here so that the layout will reflect the final generated code.